### DISCUSSION TITLE
College not AICTE approved

### DISCUSSION THREAD

**Discussion by YashTrivedi21** [github/india#155] [Students]
What if my college isn't AICTE approved? Can I still participate?

**Comment by scmishra-cse** [comment]
Hi, @YashTrivedi21 That's totally okay if your college is not in the drop-down list of AICTE internship registration, there is an option for Non-AICTE colleges registrations you can go through it...  EVEN IF YOU DIDN'T GET THE REGISTER BUTTON ACTIVE that's a common issue just refresh the page and try again.

> What if my college isn't AICTE approved? Can I still participate?

**Reply by YashTrivedi21** [reply]
Thank you very much for the clear reply!
